www/pfSense-pkg-squid/Makefile

```diff
@@ -2,7 +2,7 @@
 
 PORTNAME=	pfSense-pkg-squid
 PORTVERSION=	0.4.36
-PORTREVISION=	1
+PORTREVISION=	2
```

Comment:
package is currently in PORTREVISION 2, could you please rebase your fork and bump it to 3?